orchestrator/test_runner/src/main.rs

```diff
@@ -465,6 +469,20 @@ pub async fn main() {
             )
             .await;
             return;
+        } else if test_type == "ETHEREUM_KEYS" || test_type == "ETHERMINT_KEYS" {
+            info!("Starting Ethereum Keys test");
+            let result = ethereum_keys_test(
+                &web30,
+                grpc_client,
```

Comment:
please document this in the comment on top of the function.

Reply:
Documented